src/components/UI/Icons/Wrong.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
Does this belong in this PR? Adding correct/wrong applies to all types of questions, no?